docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -875,6 +887,89 @@ To process the received search command on the server side, follow these steps:
+   ret = mc_server_set_playback_ability(g_mc_server, MC_PLAYBACK_ACTION_FAST_FORWARD, MC_ABILITY_SUPPORTED_NO);
+   ret = mc_server_set_playback_ability(g_mc_server, MC_PLAYBACK_ACTION_REWIND, MC_ABILITY_SUPPORTED_YES);
+   ret = mc_server_set_playback_ability(g_mc_server, MC_PLAYBACK_ACTION_TOGGLE_PLAY_PAUSE, MC_ABILITY_SUPPORTED_NO);
+   
+   ret = mc_server_update_playback_ability(g_mc_server);
+   ```
+   
+   To update the display mode ability, set the ability using 'mc_server_set_display_mode_ability()':
+   ```
+   ret = mc_server_set_display_mode_ability(g_mc_server, MC_DISPLAY_MODE_LETTER_BOX | MC_DISPLAY_MODE_ORIGIN_SIZE | MC_DISPLAY_MODE_FULL_SCREEN | MC_DISPLAY_MODE_CROPPED_FULL, MC_ABILITY_SUPPORTED_YES);
+   ```
+   
+   To update the display rotation ability, set the ability using 'mc_server_set_display_rotation_ability()':
+    ```
+   ret = mc_server_set_display_rotation_ability(g_mc_server, MC_DISPLAY_ROTATION_NONE | MC_DISPLAY_ROTATION_90 | MC_DISPLAY_ROTATION_180 | MC_DISPLAY_ROTATION_270, MC_ABILITY_SUPPORTED_YES);
+   ```
+   
+   In case of other abilities, set the ability using 'mc_server_set_ability_support()', to update an ability:
+   For example, to update shuffle and repeat ability, set the ability using the 'mc_server_set_ability_support()':
+   ```
+   ret = mc_server_set_ability_support(g_mc_server, MC_ABILITY_SHUFFLE, MC_ABILITY_SUPPORTED_YES);
+   ret = mc_server_set_ability_support(g_mc_server, MC_ABILITY_REPEAT, MC_ABILITY_SUPPORTED_NO);
+   ```
+
+To retrieve the abilities on the client side, follow these steps:
+
+1. Create the media controller client handle using the `mc_client_create()`:
+
+   ```
+   ret = mc_client_create(&g_client_h);
+   ```
+
+2. Define the callback that gets invoked when the client receives the change in abilities.
+   To define a callback for a playback ability:
+   ```
+   ret = mc_client_set_playback_ability_updated_cb(g_client_h, _mc_playback_ability_updated_cb, NULL);
+   ```
+   
+   - `mc_client_set_playback_ability_updated_cb()`: For playback ability change.
+   - `mc_client_set_display_mode_ability_updated_cb()`: For display mode ability change.
+   - `mc_client_set_display_rotation_ability_updated_cb()`: For display rotation ability change.
+   - `mc_client_set_ability_support_updated_cb()`: For other abilities change.
+   
+3. Or, client can get the server's ability directly.
+   For example, to get a server's playback ability:
+   ```
+   mc_playback_ability_h ability = NULL;
+   ret = mc_client_get_server_playback_ability(g_client_h, "server_name", &ability);
+   ```
+
+   - `mc_client_get_server_display_mode_ability()`: For display mode ability.
+   - `mc_client_get_server_display_rotation_ability()`: For display rotation ability.
+   - `mc_client_get_server_ability_support()`: For other abilities.
+   
+4. Destroy the media controller client handle using the `mc_client_destroy()`, when media controller client handle is no longer needed:
```

Comment:
4. Destroy the media controller client handle using `mc_client_destroy()`,  when media controller client handle is no longer needed:

Reply:
Done